spec/index.html

```diff
@@ -2674,8 +2674,15 @@ <h2>Serialization</h2>
 <section id="privacy-considerations" class="informative">
   <h2>Privacy Considerations</h2>
 
-  <section>
-    <h3>Selective Disclosure Schemes</h3>
+  <p>In general, RDF datasets are used for representing and conveying arbitrary information,
+    which may include personally identifiable information which may be used for correlating
```

Comment:
```suggestion
    which may include personally identifiable information that may be used for correlating
```

(only stylistic change)